From bfe756dbcd79a2b34884df07462a3886b17424db Mon Sep 17 00:00:00 2001
From: r2 <r2@d2.clam.moe>
Date: Fri, 15 Aug 2025 12:01:42 +0800
Subject: [PATCH 4/4] giving up use builtin to tell strl* exists

---
 include/osdep.h | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/include/osdep.h b/include/osdep.h
index ed8bbd2a..55742f0b 100644
--- a/include/osdep.h
+++ b/include/osdep.h
@@ -18,13 +18,6 @@
 #include <sys/file.h>      /* for flock() */
 #include <strings.h>       /* for strcasecmp() */
 
-#if !defined(__has_builtin) || !__has_builtin(strlcpy)
-#define NEED_STRLCPY
-#endif
-#if !defined(__has_builtin) || !__has_builtin(strlcat)
-#define NEED_STRLCAT
-#endif
-
 #define NEED_SETPROCTITLE
 
 #elif ! defined(__FreeBSD__) || ! defined( DARWIN )
-- 
2.47.2

